yellowbrick/__init__.py

```diff
@@ -31,6 +31,7 @@
 
 # Import yellowbrick functionality to the top level
 from .anscombe import anscombe
+from .classifier import ROCAUC, ClassBalance, ClassificationScoreVisualizer
```

Comment:
I actually used it like this: 
`yb.classifier.ClassBalance(model, classes=city_options)`

I can't remember if I also tried importing i.e. from yellowbrick.classifier import ClassBalance (maybe tried it and abandoned it?)

Agree we'll hoist to second level. My opinion would be that scikit-learn has too many levels and could benefit from more hoisting, but that separate groups for features, classifiers, and regressors is a good package level to bring each visualizer to - the level defines the meaning of what it would be used for, and then you can put all the visualizer options into the root of that usage level.